src/main/java/graphql/schema/PropertyFetchingImpl.java

```diff
@@ -50,14 +53,27 @@ private class CachedMethod {
 
     }
 
+    private static class CachedFunction {
+        Function<Object, Object> getter;
```

Comment:
I remember reading that declaring fields `final` might improve method reference performance so it might apply here. Although maybe it was just `static final` constants.